utils/unittest/epoch_builder.go

```diff
@@ -374,5 +381,48 @@ func (builder *EpochBuilder) addBlock(block *flow.Block) {
 		require.NoError(builder.t, err)
 	}
 
-	builder.blocks[block.ID()] = block
+	builder.blocksByID[block.ID()] = block
+	builder.blocks = append(builder.blocks, block)
+}
+
+// AddBlocksWithSeals for the n number of blocks specified this func
+// will add a seal for the second highest block in the state and a
+// receipt for the highest block in state to the given block before adding it to the state.
+// NOTE: This func should only be used after BuildEpoch to extend the commit phase
+func (builder *EpochBuilder) AddBlocksWithSeals(n int, counter uint64) *EpochBuilder {
+	for i := 0; i < n; i++ {
+		// given the last 3 blocks in state a <- b <- c
+		// the new block will contain
+		// - seal for c
+		// - execution receipt for b
+		// - execution result for a
```

Comment:
This looks outdated - we only need to look back two blocks, and we should be including the receipt and result for the same block (no need to separate them).

Reply:
https://github.com/onflow/flow-go/pull/1940/commits/6e64634d3c34f3e167f62d74e1c1cf5e49f5d740